### ISSUE TITLE
post_mean sometimes is close to lam_mean

### ISSUE THREAD

**Issue by HilbertHuangHitomi** [zhd96/pi-vae#2]
Hi there! Thank you for sharing the code. Recently, I would like to extend the framework to analyze neural dynamics. I try to reproduce your code in PyTorch. Sometimes post_mean chart is close to and even the same as the lam_mean. The posterior latent simply repeats the behavior of p(z|u) and does not learn anything from x. I have tried to balance the loss, but it may damage the identifiability. What would be the cause? How can I fix it? Thanks!

**Comment by zhd96** [comment]
Hi, I'm not sure what cause this. You can check the posterior latent plots in our paper, they are not close to the prior lam_mean. I would suggest you to double check the loss function implementation.
